crop.py

```diff
@@ -43,7 +43,7 @@
     png = filename
     filename = filename[0:len(filename) - len(".png")] + '.json'
     needle = json.loads("""{
-        "tags": [ "FIXME" ],
+        "tags": [ "none" ],
```

Comment:
Considering the commit title "Delete obsolete FIXME *comments*" I would refrain from doing that change here.

Reply:
Yes, good point. I will extract these changes into a separate commit stating that we want to exclude the special word from template strings

Reply:
done with https://github.com/os-autoinst/os-autoinst/pull/1377/commits/1daacbbcba7b275b018cc4390e6ad858007f4642